### PR TITLE
new landing page design

### PR DESCRIPTION
**Pull request by KimberleyCook**

### CODE REVIEW

File: app/CustomFields/product-collection-acf.php [hunk 1]
```diff
@@ -17,16 +17,23 @@
         )
       ),
       array (
-        'key' => $key . '_header_image',
-        'label' => 'Product Collection Header Image',
-        'name' => 'product_collection_header_image',
+        'key' => $key . '_background_image',
+        'label' => 'Product Collection Main Background Image',
+        'name' => 'main_background_image',
```
Comment (ecoad) [inline app/CustomFields/product-collection-acf.php]: the `key` and `name` should be almost identical, apart from in `key` you prefix with `$key . '...'.

example:

```
$key = 'fancy_feature'

array(
  'key' => $key . '_this_is_my_field'
  ...
  'name' => `this_is_my_field`
)
```

File: app/CustomFields/product-collection-acf.php [hunk 2]
```diff
@@ -17,16 +17,23 @@
...
         'type' => 'image',
         'required' => 0,
         'wrapper' => array (
           'width' => '50%',
         )
       ),
       array (
+        'key' => $key . '_landing_sell',
```
Comment (ecoad) [inline app/CustomFields/product-collection-acf.php]: Check `key` and `name` match (see comment above)

**Comment by KimberleyCook** [comment]
screen shot 2016-08-24 at 17 39 17

I know the images are wrong but, I don't have either but this shows what it looks like

**Comment by thomasdigby** [comment]
Just a couple of things:
- Can we get the Best Beauty as an SVG?
- Delete the logo top left
- Update the buttons to a pink background-color with white text
- We need to sit the `product_collection_landing` over the border, maybe something like this could work?

``` css
.product_collection_landing {
  width: calc(100% + 20px);
  margin-left: -10px;
  height: 100vh;
  margin-top: -10px;
}
```

**Comment by KimberleyCook** [comment]
just seen the new designs ha

**Comment by KimberleyCook** [comment]
what is wrong with best beauty being a font?

**Comment by thomasdigby** [comment]
It will render differently cross-browser with a font, it should be consistent. Let's do that later.

**Pull request merged**